src/shared/components/Header/ui/Notification/Notification.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  ActionIcon,
+  Avatar,
+  Flex,
+  Indicator,
+  Menu,
+  Stack,
+  Text,
+  Tooltip,
+  useMantineColorScheme,
+  useMantineTheme
+} from '@mantine/core';
+import { IconBell } from '@tabler/icons-react';
+
+import { ICON_SIZE, NOTIFICATIONS } from './data';
+
+export const Notification = () => {
+  const { colorScheme } = useMantineColorScheme();
+  const theme = useMantineTheme();
+
+  const notifications = NOTIFICATIONS.slice(0, 3).map((n) => (
+    <Menu.Item
+      key={n.id}
+      style={{
+        borderBottom: `1px solid ${
+          colorScheme === 'dark' ? theme.colors.gray[7] : theme.colors.gray[3]
+        }`
```

Comment:
try use [style-prop](https://mantine.dev/styles/styles-overview/#style-prop) with css variables and read about using [data-mantine-color-scheme-attribute](https://mantine.dev/theming/color-schemes/#data-mantine-color-scheme-attribute) or use light-dark